taojieTan/quiz/q1.py

```diff
@@ -0,0 +1,52 @@
+'''
+Problem Statement
+
+Given an array of positive numbers and a positive number ‘k,’ 
+find the maximum sum of any contiguous subarray of size ‘k’.
+'''
+
+# input: an array with positive integers, a positive integer k  (k should be greater or equal to length of the array)
+# output: a maximum sum
+# edge case: k equals length of array => return sum of array
```

Comment:
Nice edge case. Another edge case to think about is if k is greater than the length of the array.